stat/running/running.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package running
+
+import "math"
+
+// Mean helps compute running mean
```

Comment:
`// Mean is a running mean accumulator.`? The "helps" seems odd.

Reply:
Done.